Cloudinary/Operation/01_ManageCloudinaryFiles.md

```diff
@@ -0,0 +1,182 @@
+    Enter a description for the image which is indicated when the image cannot be displayed. This field is automatically created for each image.
+  - *Title (caption)*   
+    Enter a title for the image. This field is automatically created for each image.
+  - *Filename*   
+    The filename is automatically adopted from the uploaded or imported image. This field is read-only. To edit a filename after uploading, open the corresponding image in Cloudinary.
+  - *Contextual meta data fields*   
+    Enter an appropriate value in the contextual meta data fields specified in the Cloudinary attribute configuration.
+  - *Structured meta data fields*   
+    Enter an appropriate value in the structured meta data fields enabled in the Cloudinary attribute configuration.
+
+    > [Info] Currently only structured meta data fields of the text and the date field type are supported by Actindo. Note to enter a date value in the appropriate date format (yyyy-MM-dd).
+
+    ![Cloudinary image settings](../../Assets/Screenshots//PIM/Products/List/Attributes/CloudinaryImageSettings.png "[Cloudinary image settings]")
+
+5. Click the [SAVE] button in the upper right corner of the *Edit product* view.    
+  The images have been saved and uploaded to Cloudinary. The *Edit product* view is closed.
+
+
+
+## Import a Cloudinary file to PIM
+
+Import images and videos from Cloudinary to products in your *PIM* module.
+
+#### Prerequisites
+
+- A Cloudinary connection has been established, see [Establish a connection to Cloudinary](../Integration/01_EstablishConnection.md)
+- The required Cloudinary attribute has been created, see [Manage the Cloudinary attributes](../Integration/02_ManageCloudinaryAttributes.md)
+- At least one file is available for import in Cloudinary.
+
+#### Procedure
+
+*PIM > Products > Tab LIST*
+
+![Products](../../Assets/Screenshots/PIM/Products/List/Products.png "[Products]")
+
+> [Info] The following procedure describes exemplary how to import an image from Cloudinary. The procedure is largely identical to import a video from CLoudinary. The differences are explicitly specified at the appropriate place.
+
+1. Click the product in the product list to which you want to add an image from Cloudinary.   
+  The *Edit product* view is displayed.
+
+  ![Edit product](../../Assets/Screenshots/PIM/Products/List/EditProduct.png "[Edit product]")
+
+2. Navigate to the Cloudinary image attribute in the *Attributes* tab.
+
+  ![Cloudinary attribute](../../Assets/Screenshots/PIM/Products/List/Attributes/CloudinaryAttribute.png "[Cloudinary attribute]")
+
+3. Click the [LIST UNASSIGNED ITEMS] button in the Cloudinary image attribute box to select the image(s) to be imported. When importing a video from Cloudinary, use the button in the corresponding Cloudinary video attribute box instead of the Cloudinary image attribute box.        
+  The *Unassigned items* window is displayed, listing all unassigned images in the specified Cloudinary folder.
+
```

Comment:
s.o. image attribute box vs. video attribute box (nicht angezeigt in Screenshot)